ui/src/literature/components/DOILinkAction.jsx

```diff
@@ -10,14 +12,11 @@ class DOILinkAction extends Component {
     const href = `//doi.org/${doi}`;
     return (
       <EventTracker eventId="DoiLink">
```

Comment:
now all even trackers should directly wrap `Button` or `Link` whatever is inside.